docs/doc/00-overview/index.md

```diff
@@ -124,28 +133,26 @@ For efficient pruning, Databend also creates indexes for each Parquet file:
 With the indexes, we can speed up the queries by reducing the I/O and CPU costs.
 Imagine that Parquet file f1 has `min_max.idx` of `[3, 5)` and Parquet file f2 has `min_max.idx` of `[4, 6)` in column `x` if the query predicate is `WHERE x < 4`, only f1 needs to be accessed and processed.
 
-## Getting Started
-
-- [Guides](/doc/guides)
+## Learn More
```

Comment:
`Community|Roadmap` and others are not `Learn More`, they are on the same level as `Databend Architucture` , and prefer to keep the title as before.